### DISCUSSION TITLE
Deployments stuck in Queued

### DISCUSSION THREAD

**Discussion by eindaniels** [vercel/community#2792] [Help]
### Summary

So, I was trying to deploy my project into Vercel with github and its stuck now in that Queue thing I dont know what to do, like do I just wait till it works?

### Steps to Reproduce

_it just happened lol_

**Comment by swarnava** [comment]
Hi, Can you share the deployment url?

**Reply by eindaniels** [reply]
here: https://vercel.com/dalol-de/ca-wiki/F8RQGTDMHeeAT4vic1hyismkEBHT
or: https://ca-wiki-kcdc4959x-dalol-de.vercel.app/

**Comment by eindaniels** [comment]
solved
